uber/templates/emails/reg_workflow/attendee_confirmation.html

```diff
@@ -3,8 +3,16 @@
 <body>
 
 You ({{ attendee.full_name }}) have preregistered for {{ c.EVENT_NAME }} this coming {{ event_dates() }}
+{% if attendee.promo_code_id %}
```

Comment:
I figured that it'd be nice to print out the promo code they used even if their badge isn't free, so that's a separate block here.